cmd/lotus-fountain/main.go

```diff
@@ -193,12 +195,30 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	to, err := address.NewFromString(r.FormValue("address"))
-	if err != nil {
+	var to address.Address
```

Comment:
We're not setting this anywhere, and using for the send destination, probably want to remove this var, and swap uses of `to` to `filecoinAddress`